kymatio/scattering3d/backend/torch_backend.py

```diff
@@ -59,10 +25,9 @@ def modulus_rotation(x, module=None):
             which is covariant to 3D translations and rotations.
     """
     if module is None:
-        module = torch.zeros_like(x)
+        module = (x ** 2).sum(-1).unsqueeze(-1)
```

Comment:
Check if `torch.sum` has a `keepdims` argument and use that if so, otherwise `[..., np.newaxis]`

Reply:
Yes it does, I can use that.